.travis.yml

```diff
@@ -16,11 +16,13 @@ before_install:
     - git clone --quiet --depth 1 https://github.com/minrk/travis-wheels travis-wheels
 install:
     - wget http://c90cb902a90d6a506ab3-a24171cbeaa42ce11788f0df3362e902.r46.cf5.rackcdn.com/pandoc-1.15.0.6.tar.gz && tar -xzf pandoc-1.15.0.6.tar.gz
-    - pip install -f travis-wheels/wheelhouse . coveralls codecov
+    - pip install -f travis-wheels/wheelhouse . codecov coverage
     - pip install nbconvert[execute,serve,test]
     - python -m ipykernel.kernelspec --user
 script:
-    - nosetests -w /tmp --with-coverage --cover-package nbconvert nbconvert
+    - # need to cd or coverage report broken.
+    - cd /tmp/
+    - nosetests --with-coverage --cover-package nbconvert --cover-inclusive --cover-xml nbconvert --cover-xml-file=$TRAVIS_BUILD_DIR/coverage.xml
```

Comment:
args thing fixed. It was not working locally without --cover-inclusive, but I can re-try.